fpo-api/api/send_email.py

```diff
@@ -47,6 +47,13 @@ def send_email(recipient_email: str, pdf_data: bytes):
 
     LOGGER.info("Recipient email address: %s", recipient_email)
 
+    if (pdf_name == "violation-ticket-statement-and-written-reasons"):
```

Comment:
Conditionals don't get extra brackets around them in python. You can run Black to autoformat the code, or VS code will do it for you.